extensions/ql-vscode/src/view/jest.setup.ts

```diff
@@ -16,14 +16,12 @@ Object.defineProperty(window, "matchMedia", {
 });
 
 // Used by Primer React
-(window as any).CSS = {
-  supports: jest.fn().mockResolvedValue(false),
-};
+window.CSS.supports = jest.fn().mockResolvedValue(false);
```

Comment:
That is a good point that it is a different behaviour from before. I forgot/missed that when I wrote up the PR description.

When I just changed the type this line was producing an error because the `CSS` object contains other fields so just overwriting it with `{ supports }` wasn't satisfying the full type. That's why I changed it to just overwrite the `supports` field instead of the entire `CSS` object.

As you say, since it was passing before and also passing now, I suppose it doesn't matter. The `CSS` field exists already, and apparently we don't use any other fields from it because it was ok when we overwrote it to a nearly-empty object.